test/e2e/cluster_configs/vmas.json

```diff
@@ -0,0 +1,36 @@
+{
+    "apiVersion": "vlabs",
+    "properties": {
+        "orchestratorProfile": {
+            "orchestratorType": "Kubernetes",
+            "orchestratorRelease": "$K8S_RELEASE"
+        },
```

Comment:
Let's also add the addon `aad-pod-identity` and set to false explicitly in the api model.